packages/sources/src/components/definition-list/definition-list.scss

```diff
@@ -183,6 +195,26 @@ $dso-definition-list-highlight-color: $grasgroen;
       }
     }
 
+    &.dl-horizontal {
+      dd {
+        @include clearfix;
```

Comment:
Is deze nog nodig? Op regel 41 wordt ie ook al gezet

(En daarnaast.. `()` ?)

Reply:
![image](https://user-images.githubusercontent.com/9696261/122392972-ac5f1300-cf74-11eb-9384-b04e906723f7.png)

Dat dus